Sizzle/Sizzle/Sizzle/recipeView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  recipeView.swift
+//  Sizzle
+//
+//  Created by Charlie Nevitt on 11/2/22.
+//
+
+import SwiftUI
+
+struct recipeView: View {
+    let image:String
+    let recipeName: String
+    @State private var selectedTab = "one"
+    var body: some View {
+        VStack{
+            Image(image)
+            Text(recipeName)
+            TabView(selection:$selectedTab){
+                Text("Ingredients")
+                    .onTapGesture {
```

Comment:
This works, but generally this should probably be a `Button` where the text is the button's label. Sometimes a tapGesture is appropriate but I don't see the need here.